### PR TITLE
Add account admin overview section

### LINKED ISSUE #1199: Modify the privacy page to support the new account page
### Feature description

From design:

Screenshot 2022-06-21 at 18 02 52

We are going to need to create a new account page where it was the privacy page. This will require a little structure refactorization plus adding the new `Account` section.

- [x] Modify the privacy page to includes the new `Account` section
- [x] Connect this new section to save the API_KEY with the backend
- [x] URL refactorizaction: right now this page lives on `/account/privacy`. Look for a new name for the URL.

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Closes #1199 

## Implementation details

- rename `PrivacySection` component to `AccountAdmin`
- updated all related classes
- add `userData.email` as prop to pass down to `AccountAdmin`
- add `BXInput`
- include strings for translation 
- added functionality for saving and copying tokens

## Screenshots

https://user-images.githubusercontent.com/6276074/179068874-c0c67c8a-b5ce-47d1-8d97-e633419b0bfd.mov

### CODE REVIEW

**Comment by Tansito** [comment]
@techtolentino I added the steps to configure the API Token, let me know what you think. I didn't put the PR as `Ready for review` because I saw that we need to prepare some kind of feedback for the user and some `console.logs` around but let me know!

**Comment by Tansito** [comment]
It seems that everything is ready for review so I open the PR @techtolentino !

File: frontend/vue/components/UserAccount/AccountAdmin.vue [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    & > ::v-deep(*):last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__policy-email {
+    color: $link-color-secondary;
+  }
+}
+
```
Comment (eddybrando) [inline frontend/vue/components/UserAccount/AccountAdmin.vue:288]: Just removing this unnecessary blank line 😬:

```suggestion
```

File: frontend/vue/components/UserAccount/AccountAdmin.vue [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    top: 1.35rem;
+    right: 0.15rem;
+    height: 2.75rem;
+    width: 2.75rem;
+    background-color: $background-color-lighter;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    &:hover {
```
Comment (eddybrando) [inline frontend/vue/components/UserAccount/AccountAdmin.vue:241]: Adding a blank line for better readability:

```suggestion

    &:hover {
```

**Pull request merged**